portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,12 +62,18 @@ private String toJsonUsingGson(List<String> data) {
         return json;
     }
 
-    /** Process POST request form input, add to pastCommens, and redirect to index.html */
+    /** Process POST request form input, add to pastComments, and redirect to index.html */
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Get the input from the form.
         String comment = getParameter(request, "text-input", "");
-        pastComments.add(0, comment);
+        // pastComments.add(0, comment);
```

Comment:
Don't leave behind commented out code unless it has a purpose.  If it does have a purpose, at a comment at the top of the commented out code explaining why it's there.

Reply:
Got it, will do